coops-ui/src/components/AnnualReport/Directors.vue

```diff
@@ -849,6 +932,16 @@ export default class Directors extends Mixins(DateMixin, ExternalMixin) {
         addressCountry: this.inProgressAddress.addressCountry,
         deliveryInstructions: this.inProgressAddress.deliveryInstructions
 
+      },
+      mailingAddress: {
+        streetAddress: this.inProgressMailAddress.streetAddress,
+        streetAddressAdditional: this.inProgressMailAddress.streetAddressAdditional,
+        addressCity: this.inProgressMailAddress.addressCity,
+        addressRegion: this.inProgressMailAddress.addressRegion,
+        postalCode: this.inProgressMailAddress.postalCode,
+        addressCountry: this.inProgressMailAddress.addressCountry,
+        deliveryInstructions: this.inProgressMailAddress.deliveryInstructions
+
```

Comment:
BTW, I think we should initialize address type here (and also above). It may break other code but I think it's the correct thing to do (and fix).

Reply:
Obsolete comment. We have instead decided to delete/not use the addressType property.